### PR TITLE
Dịch introduction/index_vn.md phần 28

### LINKED ISSUE #178: Dịch introduction/index_vn.md phần 28
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/943fd8472a3ec07dff69baa53ef093487dc9050c/chapter_introduction/index_vn.md#L1864-L1942

### PR DESCRIPTION
**Pull request by quangnhat185**
## Pull Request checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #178 

* [x] Bản dịch này có bám sát [Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md) không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm
    * Tiêu đề (số lượng dấu `#` đầu dòng)
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`)
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt)
    * Không dịch các phần code (nằm giữa hai dấu `````)
    * Copy các công thức toán từ bản gốc (các đoạn có `$`)
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`)
    * Không tự thêm bớt các dòng trắng
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm [hướng dẫn hoặc yêu cầu dịch hình](../img/README.md).

* [x] Điền tên của bạn và vào mục "Những người thực hiện" ở cuối file.

* [x] Pull Request này đã sẵn sàng cho việc Review chưa?

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này.

## Với reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

**Comment by duythanhvn** [comment]
Nhật vào Collaborator rồi mà Labeler check vẫn lỗi sao ta.

File: chapter_introduction/index_vn.md [hunk 1]
```diff
@@ -2220,7 +2222,14 @@ and diminish undesirable behavior to obtain good settings
 of the parameters in a neural network.
 -->
 
-*dịch đoạn phía trên*
+Một ảnh hưởng khác có thể được tìm thấy trong khoa học thần kinh và tâm lý học.
+Sau cùng, loài người thể hiện rất rõ ràng bản năng thông minh của mình.
+Sẽ là hợp lý nếu chúng ta có thể giải thích và phân tích ngược khả năng này.
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Tự nhiên là sẽ có người hỏi liệu chúng ta có thể giải thích và tái tạo lại sự thông minh này.
```

File: chapter_introduction/index_vn.md [hunk 2]
```diff
@@ -2220,7 +2222,14 @@ and diminish undesirable behavior to obtain good settings
...
+Một trong những thuật toán lâu đời nhất được truyền cảm hứng từ hiện tượng đó được giới thiệu bởi [Donald Hebb (1904-1985)](https://en.wikipedia.org/wiki/Donald_O._Hebb).
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Một trong những thuật toán lâu đời nhất lấy cảm hứng từ câu hỏi trên được giới thiệu bởi [Donald Hebb (1904-1985)](https://en.wikipedia.org/wiki/Donald_O._Hebb).
```

File: chapter_introduction/index_vn.md [hunk 3]
```diff
@@ -2220,7 +2222,14 @@ and diminish undesirable behavior to obtain good settings
...
+Trong cuốn sách đột phát của mình về Tổ Chức Hành Vi :cite:`Hebb.Hebb.1949`, ông ấy cho rằng các nơ-ron học bằng sự tăng cường tích cực.
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Trong cuốn sách đột phá của mình về Tổ Chức Hành Vi :cite:`Hebb.Hebb.1949`, ông ấy cho rằng các nơ-ron học bằng sự tăng cường tích cực.
```

File: chapter_introduction/index_vn.md [hunk 4]
```diff
@@ -2220,7 +2222,14 @@ and diminish undesirable behavior to obtain good settings
...
+Điều này về sau được biết đến với cái tên là luật học Hebbian.
+Nó là nguyên mẫu từ thuật toán perceptron của Rosenblatt và là nền tảng của rất nhiều thuật toán hạ gradient ngẫu nhiên đã đặt nền móng cho học sâu ngày nay:
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Nó là nguyên mẫu cho thuật toán perceptron của Rosenblatt và là nền tảng của rất nhiều thuật toán hạ gradient ngẫu nhiên đã đặt nền móng cho học sâu ngày nay:
```

File: chapter_introduction/index_vn.md [hunk 5]
```diff
@@ -2232,14 +2241,17 @@ but the name stuck. At its heart, lie a few key principles
 that can be found in most networks today:
 -->
 
-*dịch đoạn phía trên*
+Niềm cảm hứng từ sinh học đã tạo nên cái tên *Mạng nơ-ron*.
+Trong suốt hơn một thế kỷ (trở lại với mô hình của Alexander Bain, 1873 và James Sherrington, 1890), các nhà ngiên cứu đã cố gắng lắp ráp các mạch tính toán nhằm tạo ra những mạng lưới nơ-ron tương tác.
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Trong suốt hơn một thế kỷ (từ mô hình của Alexander Bain, 1873 và James Sherrington, 1890), các nhà ngiên cứu đã cố gắng lắp ráp các mạch tính toán tương tự như các mạng lưới nơ-ron tương tác.
```

File: chapter_introduction/index_vn.md [hunk 6]
```diff
@@ -2232,14 +2241,17 @@ but the name stuck. At its heart, lie a few key principles
...
+Theo thời gian, yếu tố sinh học ngày càng giảm đi nhưng tên gọi của nó thì vẫn ở lại. Những nguyên tắc thứ yếu của mạng nơ-ron vẫn có thể tìm thấy ở hầu hết các mạng ngày nay:
 
 <!--
 * The alternation of linear and nonlinear processing units, often referred to as *layers*.
 * The use of the chain rule (also known as *backpropagation*) for adjusting parameters in the entire network at once.
 -->
 
-*dịch đoạn phía trên*
+* Sự đan xen giữa các đơn vị xử lý tuyến tính và phi tuyến tính, thường được đề đề cập tới như là *các tầng*.
+* Việc sử dụng quy tắc chuỗi (còn được biết đến là *backpropagation*) để điều chỉnh các tham số trong toàn bộ mạng cùng lúc.
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
* Việc sử dụng quy tắc chuỗi (còn được biết đến là *lan truyền ngược -- backpropagation*) để điều chỉnh các tham số trong toàn bộ mạng cùng lúc.
```

File: chapter_introduction/index_vn.md [hunk 7]
```diff
@@ -2254,7 +2266,10 @@ was a popular tool for testing the efficacy of algorithms.
 MNIST with its 60,000 handwritten digits was considered huge.
 -->
 
-*dịch đoạn phía trên*
+Sau những tiến bộ nhanh chóng ban đầu, các nghiên cứu về mạng nơ-ron giảm dần trong khoảng từ 1995 tới 2005 bởi một vài lí do.
+Thứ nhất là huấn luyện mạng rất tốt kém tài nguyên tính toán.
+Mặc dù lượng RAM đã dồi dào vào cuối thế kỷ trước, sức mạnh tính toán vẫn còn hạn chế.
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Mặc dù dung lượng RAM đã dồi dào vào cuối thế kỷ trước, sức mạnh tính toán vẫn còn hạn chế.
```

File: chapter_introduction/index_vn.md [hunk 8]
```diff
@@ -2254,7 +2266,10 @@ was a popular tool for testing the efficacy of algorithms.
...
+Thứ hai là tập dữ liệu vẫn còn tương đối nhỏ. Lúc đó tập dữ liệu Fisher's Iris từ năm 1932 vẫn là công cụ phổ biến để kiểm tra tính hiệu quả của các thuật toán. MNIST với 60,000 ký tự viết tay được xem là rất lớn
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Thứ hai là tập dữ liệu vẫn còn tương đối nhỏ. Lúc đó tập dữ liệu Fisher's Iris từ năm 1932 vẫn là công cụ phổ biến để kiểm tra tính hiệu quả của các thuật toán. MNIST với 60,000 ký tự viết tay đã được xem là rất lớn.
```

File: chapter_introduction/index_vn.md [hunk 9]
```diff
@@ -2264,7 +2279,8 @@ Unlike neural networks, they did not require weeks to train
 and provided predictable results with strong theoretical guarantees.
 -->
 
-*dịch đoạn phía trên*
+Với sự khan hiếm của dữ liệu và tài nguyên tính toán, các công cụ thống kê mạnh như Phương Pháp Kernel, Cây Quyết Định và Mô Hình Dạng Đồ Thị đã chứng tỏ sự vượt trội trong thực nghiệm.
```
Comment (lkhphuc) [inline chapter_introduction/index_vn.md]:
```suggestion
Với sự khan hiếm của dữ liệu và tài nguyên tính toán, các công cụ thống kê mạnh như Phương Pháp Hạt, Cây Quyết Định và Mô Hình Dạng Đồ Thị đã chứng tỏ sự vượt trội trong thực nghiệm.
```
kernel method hình như a @tiepvupsu dịch là phương pháp hạt hay hạt nhân đúng không, để em cho vô glossary luôn.

Comment (lkhphuc) [reply chapter_introduction/index_vn.md]: Graphical model chắc sẽ chờ #235 luôn.

File: chapter_introduction/index_vn.md (hunk 1 above)
Comment (quangnhat185) [reply chapter_introduction/index_vn.md]: Đoạn này mình vẫn thấy có chút gì đó chưa thoát ý. Để triệu hồi 2 trùm cuối @ngcthuong và @tiepvupsu vào bàn thêm. Em để bản gốc ở đây luôn để các anh khỏi mất công roll ngược lên: 

> Another influence can be found in neuroscience and psychology. After all, humans clearly exhibit intelligent behavior. **It is thus only reasonable to ask whether one could explain and possibly reverse engineer this capacity**.

File: chapter_introduction/index_vn.md (hunk 9 above)
Comment (tiepvupsu) [reply chapter_introduction/index_vn.md]: @lkhphuc sách anh dịch là "phương pháp hạt nhân".

"Graphical model" anh thấy dịch là "mô hình đồ thị" thì ổn.

Từ "graphical" như anh đã giải thích ở đâu đó, nó là tính từ của "graph". Mà "graph" vẫn được dịch là "đồ thị" rồi.

File: chapter_introduction/index_vn.md [hunk 10]
```diff
@@ -2232,14 +2241,17 @@ but the name stuck. At its heart, lie a few key principles
 that can be found in most networks today:
 -->
 
-*dịch đoạn phía trên*
+Niềm cảm hứng từ sinh học đã tạo nên cái tên *Mạng nơ-ron*.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Niềm cảm hứng từ sinh học đã tạo nên cái tên *mạng nơ-ron*.
```

File: chapter_introduction/index_vn.md [hunk 11]
```diff
@@ -2232,14 +2241,17 @@ but the name stuck. At its heart, lie a few key principles
...
+Trong suốt hơn một thế kỷ (từ mô hình của Alexander Bain, 1873 và James Sherrington, 1890), các nhà ngiên cứu đã cố gắng lắp ráp các mạch tính toán tương tự như các mạng lưới nơ-ron tương tác.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Trong suốt hơn một thế kỷ (từ mô hình của Alexander Bain, 1873 và James Sherrington, 1890), các nhà ngiên cứu đã cố gắng lắp ráp các mạch tính toán tương tự như các mạng tương tác giữa các nơ-ron.
```

File: chapter_introduction/index_vn.md [hunk 12]
```diff
@@ -2232,14 +2241,17 @@ but the name stuck. At its heart, lie a few key principles
...
+Theo thời gian, yếu tố sinh học ngày càng giảm đi nhưng tên gọi của nó thì vẫn ở lại. Những nguyên tắc thứ yếu của mạng nơ-ron vẫn có thể tìm thấy ở hầu hết các mạng ngày nay:
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Theo thời gian, yếu tố sinh học ngày càng giảm đi nhưng tên gọi của nó thì vẫn ở lại.
Những nguyên tắc thứ yếu của mạng nơ-ron vẫn có thể tìm thấy ở hầu hết các mạng ngày nay:
```

👍

File: chapter_introduction/index_vn.md [hunk 13]
```diff
@@ -2254,7 +2266,10 @@ was a popular tool for testing the efficacy of algorithms.
 MNIST with its 60,000 handwritten digits was considered huge.
 -->
 
-*dịch đoạn phía trên*
+Sau những tiến bộ nhanh chóng ban đầu, các nghiên cứu về mạng nơ-ron giảm dần trong khoảng từ 1995 tới 2005 bởi một vài lí do.
+Thứ nhất là huấn luyện mạng rất tốt kém tài nguyên tính toán.
+Mặc dù dung lượng RAM đã dồi dào vào cuối thế kỷ trước, sức mạnh tính toán vẫn còn hạn chế.
+Thứ hai là tập dữ liệu vẫn còn tương đối nhỏ. Lúc đó tập dữ liệu Fisher's Iris từ năm 1932 vẫn là công cụ phổ biến để kiểm tra tính hiệu quả của các thuật toán. MNIST với 60,000 ký tự viết tay đã được xem là rất lớn.
```
Comment (tiepvupsu) [inline chapter_introduction/index_vn.md]:
```suggestion
Thứ hai là tập dữ liệu vẫn còn tương đối nhỏ. Lúc đó tập dữ liệu Fisher's Iris từ năm 1932 vẫn là công cụ phổ biến để kiểm tra tính hiệu quả của các thuật toán. MNIST với 60.000 ký tự viết tay đã được xem là rất lớn.
```

File: chapter_introduction/index_vn.md (hunk 9 above)
Comment (quangnhat185) [reply chapter_introduction/index_vn.md]:
```suggestion
Với sự khan hiếm của dữ liệu và tài nguyên tính toán, các công cụ thống kê mạnh như Phương Pháp Hạt Nhân, Cây Quyết Định và Mô Hình Đồ Thị đã chứng tỏ sự vượt trội trong thực nghiệm.
```
